src/repl-frontend/scala/tools/nsc/interpreter/shell/ILoop.scala

```diff
@@ -326,7 +326,7 @@ class ILoop(config: ShellConfig, inOverride: BufferedReader = null,
        |"Kind" is a word used to classify types and type constructors
        |according to their level of abstractness.
        |
-       |Concrete, fully specified types such as `Int` and `Option[Int]`
+       |Concrete, fully-specified types such as `Int` and `Option[Int]`
```

Comment:
So does the emperor say, "...fully-armed and -operational battle station"?

http://www.moviequotedb.com/movies/star-wars-episode-vi-return-of-the-jedi/quote_29955.html
http://www.moviequotedb.com/movies/star-wars-episode-vi-return-of-the-jedi/character_3175.html

https://www.dailywritingtips.com/adverbs-and-hyphens/

I wish I could remember why I had to google this rule not very long ago. A long time ago in a far-far-away galaxy...

Reply:
tldr; is compound with adverb in ly isn't hyphenated.

Reply:
The example reminded me, I had joked that "get up to date" means you got out of bed to meet someone for dinner. The link says hyphenation after linking verbs is inconsistent, but doesn't mention "get". "I am up-to-date" or "I am up to date". What about, "I am up to snuff"? That is what a hit man says to the bathroom mirror in the morning.

Reply:
TIL, thanks.

Reply:
Somehow, thinking of those words brought back to me memories of this: https://xkcd.com/1771/